src/main/kotlin/com/statsig/sdk/ServerDriver.kt

```diff
@@ -1,15 +1,20 @@
 package com.statsig.sdk
 
 import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.GlobalScope
 import kotlinx.coroutines.Job
+import kotlinx.coroutines.SupervisorJob
 import kotlinx.coroutines.flow.launchIn
 import kotlinx.coroutines.flow.onEach
+import kotlinx.coroutines.future.future
 import java.util.Properties
+import java.util.concurrent.CompletableFuture
 
-internal class ServerDriver(
-    private val coroutineScope: CoroutineScope,
+
+class ServerDriver(
```

Comment:
This way, you don't have to pass in a scope in the Java unit tests below.
```suggestion
class ServerDriver @JvmOverloads constructor(
```